docs/source/examples/dynamical_phase_transitions_zne.md

```diff
@@ -0,0 +1,352 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Introduction
+
+This tutorial replicates some of the results from Y. Javanmard et al., ["Quantum simulation of dynamical phase transitions in noisy quantum devices"](https://arxiv.org/abs/2211.08318). We build a circuit that simulates the the time-evolution of a transverse-field Ising model (Eq. 1) after a quench at time $t = 0$, then run ideal, noisy, and noise-mitigated simulations of the circuit.
+
+Assuming the system is in state $\psi_0 = H^{\otimes N} \ket { 0^{\otimes N} }$ at $t = 0$, we want to compute the probability of returning to the initial state at time $t$, $\Lambda(t) = \left|\bra{\psi_0}U(t)\ket{\psi_0}\right|^2$. To simulate the behavior over the interval $[0, t_{\textrm{max}}]$, we divide it into $M$ steps. Letting $\delta t = t_{\textrm{max}}/M$, we construct circuits corresponding to $U(t) = [U(\delta t)]^k$ for $k = 0, \ldots, m$.
+
++++
+
+# Circuit definition
+
```

Comment:
Ideally we want those tutorials to be self-contained, and explain the idea behind the paper without referencing figures and equations from there.